modules/util/source-formatter/src/main/java/com/liferay/source/formatter/JavaSourceProcessor.java

```diff
@@ -2202,6 +2202,18 @@ else if ((absolutePath.contains("/test/unit/") ||
 				fileName, "Do not use Registry in modules: " + fileName);
 		}
 
+		// LPS-60186
+
+		if (fileName.endsWith("Configuration.java") &&
```

Comment:
Hey @hhuijser Would it be possible to use an smarter rule for this instead of base it in the file ending with Configuration.java? I fear that such a name might be too common. The ideal check would be to look for the @Meta.OCD annotation (or the new standard alternative).

Is that doable?